flake.nix

```diff
@@ -188,13 +187,7 @@
 
           formatter = pkgs_.nixpkgs.nixpkgs-fmt;
 
-          packages = waypkgs.waylandPkgs //
-          {
-            default = (waypkgs.linkFarmFromDrvs
```

Comment:
packages.nix uses .default

Reply:
also
```
$ nix build ".#wlvncc"
error: attribute 'libvncserver_master' missing

       at /home/artturin/nixgits/my-nixpkgs-wayland/.worktree/1/flake.nix:124:30:

          123|             wlvncc = prev.callPackage ./pkgs/wlvncc {
          124|               libvncserver = final.libvncserver_master;
             |                              ^
          125|             };
(use '--show-trace' to show detailed location information)
```

Reply:
yeah. looking at that now, not really sure I understand what exactly broke it.

Reply:
probably just going to back this change out.